aer/src/lib.rs

```diff
@@ -66,7 +66,7 @@ pub mod runtime;
 #[macro_export]
 macro_rules! execute {
     ($call:path, $req:ident) => {{
-        let client = ::aurae_client::AuraeClient::default().await?;
+        let client = ::client::AuraeClient::default().await?;
```

Comment:
should this also be Client instead of AuraeClient?

Reply:
You mean ::client::Client::default().await?;

Reply:
yes. I think it should.

sorry for the rounds of comments. every time you add more files I see more 😅

Reply:
No worries. I’m having fun exploring the code base.